app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -172,6 +173,18 @@ public void setVersion(Long version) {
         this.version = version;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public boolean isAutoVerificationEmailSuppressed() {
+        return autoVerificationEmailSuppressed;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void setAutoVerificationEmailSuppressed(boolean autoVerificationEmailSuppressed) {
+        this.autoVerificationEmailSuppressed = autoVerificationEmailSuppressed;
+    }
```

Comment:
isSendVerificationEmail[OnSignUp]Enabled, to be consistent with other flags?

Reply:
The reason why I named the flag this way is because I wanted the default value to be false so we can avoid writing code to backfill all the previous studies. I could change the name from “suppressed” to “disabled”, but it would still be backwards relative to all the other flags.